vkcs/networking/data_source_vkcs_networking_network_test.go

```diff
@@ -100,8 +100,6 @@ func TestAccNetworkingNetworkDataSource_externalExplicit(t *testing.T) {
 						"data.vkcs_networking_network.network_1", "admin_state_up", "true"),
 					resource.TestCheckResourceAttr(
 						"data.vkcs_networking_network.network_1", "external", "true"),
-					resource.TestCheckResourceAttr(
```

Comment:
Is there any test cheking tags for the datasource?

Reply:
Yes, there are several of them . For example, here: https://github.com/vk-cs/terraform-provider-vkcs/blob/b1f81d0a3c4f9a9ba70d1099ffc5b7c314cee445/vkcs/networking/data_source_vkcs_networking_network_test.go#L54-L57